### ISSUE TITLE
show validation loss

### ISSUE THREAD

**Issue by felixbur** [felixbur/nkululeko#104]
currently in epoch progression plots only the training loss is shown, and this even often badly scaled

**Comment by felixbur** [comment]
done with 78.1
